client/me/pending-payments/test/index.js

```diff
@@ -85,7 +85,7 @@ describe( 'PendingPayments', () => {
 		const rules = [
 			'Main.pending-payments Localized(MeSidebarNavigation)',
 			'Main.pending-payments PurchasesHeader[section="pending"]',
-			'Main.pending-payments Localized(PendingListItem)',
+			'Main.pending-payments Connect(Localized(PendingListItem))',
 		];
 
 		rules.forEach( rule => {
```

Comment:
No biggie, but if you want to get rid of these rules and the `forEach`s, you could just check each wrapper against a snapshot, e.g., for every test:

```
	describe( 'Loading Placeholder', () => {
		const wrapper = shallow( <PendingPayments { ...defaultProps } /> );
		expect( wrapper ).toMatchSnapshot();
/*		const rules = [
			'Main.pending-payments Localized(MeSidebarNavigation)',
			'Main.pending-payments PurchasesHeader[section="pending"]',
			'Connect(PurchasesSite)[isPlaceholder=true]',
		];

		rules.forEach( rule => {
			test( rule, () => {
				expect( wrapper.find( rule ) ).toHaveLength( 1 );
			} );
		} );*/
	} );
```

Multiple snapshots will be stored in `client/me/pending-payments/test/__snapshots__/index.js.snap`